realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -444,7 +444,9 @@ public SyncUser run() {
      * @return {@code SyncUser} associated with the given identity provider and providerId, or {@code null} in case
      * of an {@code invalid} provider or {@code providerId}.
      * @throws ObjectServerError in case of an error.
+     * @deprecated  As of release 3.6.0, replaced by {@link #retrieveInfoForUser(String, String)}}
```

Comment:
Fine if we stick to this convention, it's upper case on the annotation [doc](https://docs.oracle.com/javase/8/docs/technotes/guides/javadoc/deprecation/deprecation.html) btw